modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs
+
+The Red Hat OpenShift Container Platform Web Console provides support for the Administrator and the Developer perspectives. Cluster administrators have the ability to organize and manage a Developer catalog or its sub-catalogs based on the access control and privileges provided to a user. In some situations, customers do not want their users to have access to a particular Developer catalog or it's sub-catalogs. Customizing a user's role ensures that the user only views the Developer catalog or sub-catalogs resources assigned to them.
+
+For example, if you do not want a user to access Developer catalog resources such as Applications, Operator Backed services, Databases, Helm chart repositories etc, you can customize a user's profile accordingly.
+
+Each Developer catalog includes the following parameter that must be defined in a snippet to be used in the YAML view:
+
+* `state`: Defines whether the Developer catalog is enabled or disabled
```

Comment:
* `enabled`: enabled is a list of developer catalog types (sub-catalogs) that will be shown to users.
* `disabled`: disabled is a list of developer catalog types (sub-catalogs) that are not shown to users.

Reply:
Fixed.